resources/views/pages/contract/index.blade.php

```diff
@@ -40,27 +40,27 @@
     </section>
 
     <br />
-    <div id="company" class="container">
-    <div class="row" id="company-div">
-        @foreach ($companies as $company)  
-          <a href="{{ route('contractors.single', ['company' => $company->shortname()]) }}">
+    <div id="contractor" class="container">
+    <div class="row" id="contractor-div">
+        @foreach ($contractors as $contractor)  
+          <a href="#">
             <div class="col-md-4 col-lg-3 mb-3 card-col">
               <div class="card shadow">
                 <div class="card-body">
                     <chart label="myVueChart"
-                            v-bind:data="[{amount:32424, year:2020},{amount:12920923, year:2010}]"
-                            element="{{ $company->shortname() }}"></chart>
-                    <div class="company mb-2">
-                        <img src="{{ asset('images/image 13.png') }}" height="30" class="mr-3" alt="">
-                        <a href="{{ route('contractors.single', ['company' => $company->shortname()]) }}">
-                          <h5 class="card-title mb-0" class="company-name">
-                            {{ $company->name }}
+                            v-bind:data="[{amount: {{round($contractor->total_amount / 12)}}, year: {{$contractor->year}} }, {amount:{{$contractor->total_amount}}, year:{{$contractor->year}} }]"
+                            element="{{ \Str::slug($contractor->beneficiary, '-') }}"></chart>
+                    <div class="contractor mb-2">
+                        <!-- <img src="{{ asset('images/image 13.png') }}" height="30" class="mr-3" alt=""> -->
```

Comment:
I'm thinking in future, we should have a db column for accepting the logo url for a company. Same with ministries. This should come in later.